jovian/__init__.py

```diff
@@ -20,11 +21,25 @@
 __flavor__ = get_flavor()
 
 set_notebook_name()
-
 _current_slug = None
 _data_blocks = []
 
 
+def notify_update_available():
+    """Compares the installed and latest version of the library and notifies 
+    the user when they import the library and there is a update available.
+    """
+    try:
+        latest_version = requests.get('https://pypi.org/pypi/jovian/json', timeout=3).json()['info']['version']
+        if latest_version > __version__:
+            print('Update available: {0} --> {1}'.format(__version__, latest_version))
```

Comment:
instead of printing here, you can return latest_version. we can handle it depending upon where it's being called